### ISSUE TITLE
Allow changing the language to translate to

### ISSUE THREAD

**Issue by arnaudvergnet** [mozilla/firefox-translations#357]
it would be nice to allow translating to a language different from the one set in the browser. One use case is when you do not have the same native language as friends and want to translate a website in their language.

**Comment by Miserlou** [comment]
Yes, I would expect this tool to have From and To options to both be selectable. I often need to translate things in both directions, as I'm sure anybody else who has translanguage conversations will as well.

**Comment by Miserlou** [comment]
Seems like this will only work for German for now, though: https://github.com/mozilla/firefox-translations-models

**Comment by cadeyrn** [comment]
I would also love to see this feature in Firefox translations. :)
